WordPress/src/jetpack/assets/support_chat_widget.html

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
```

Comment:
Just a note. On iOS the chat widget only took a small portion of the screen. I applied a responsive content meta tag for the chat widget to take the full width of the screen:

```
<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

When I tested on Android with Pixel 4 it appeared correctly so I'm not sure if that's necessary.

Reply:
Wouldn't hurt having it there